template-parts/header-personal.php

```diff
@@ -36,11 +36,14 @@
 				</div>
 				<div>
 					<div class="h5 bloginfo-name">
-						<p><?php printf( '<span>%s</span>', esc_attr( get_bloginfo( 'name' ) ) ); ?></p>
-					</div>
-					<div class="mb-0 bloginfo-description">
-						<?php esc_attr( bloginfo( 'description' ) ); ?>
+						<?php printf( '<span>%s</span>', esc_attr( get_bloginfo( 'name' ) ) ); ?>
 					</div>
+					<?php
+						$description = bloginfo( 'description' );
+						if ( $description !== '' ) {
```

Comment:
```suggestion
						if ( $description ) {
```
That's ok, but the coding style requires "Yoda Condition checks" (string is before the operator). Or you just remove it.

The variable must be renamed to `$sunflower_description` due to another rule.
`Global variables defined by a theme/plugin should start with the theme/plugin prefix. Found: "$description".`